launcher/firstLaunch/firstlaunch_moc.cpp

```diff
@@ -332,10 +340,10 @@ void FirstLaunchView::extractGogData()
 		return file;
 	};
 
-	QString fileBin = fileSelection("bin", tr("GOG data") + " (*.bin)");
+	QString fileBin = fileSelection("idska32", tr("GOG data") + " (*.bin)");
 	if(fileBin.isEmpty())
 		return;
-	QString fileExe = fileSelection("exe", tr("GOG installer") + " (*.exe)", QFileInfo(fileBin).absolutePath());
+	QString fileExe = fileSelection("MZ", tr("GOG installer") + " (*.exe)", QFileInfo(fileBin).absolutePath());
```

Comment:
technically speaking, we should also check for PE format: https://learn.microsoft.com/en-us/windows/win32/debug/pe-format#signature-image-only

although I think that even this solution is already good

Reply:
Thanks. Yes, was aware of this.
But I think in our case it's good enough like it is. To keep it simple.